scripts/deployment/setup_ovs.py

```diff
@@ -794,8 +794,106 @@ def install_autotests(node_ip):
     con.process.execute("pip install vnc2flv")
 
 
+def create_autotest_cfg(os_name, vmware_info, template_server, screen_capture, test_run, vpool_config, vpool_name,
+                        backend_name, ceph_vpool_info, cinder_type, grid_ip, test_project):
+    cmd = '''source /etc/profile.d/ovs.sh
+pkill Xvfb
+pkill x11vnc
+sleep 3
+Xvfb :1 -screen 0 1280x1024x16 &
+export DISPLAY=:1.0
+x11vnc -display :1 -bg -nopw -noipv6 -no6 -listen localhost -xkb  -autoport 5950 -forever
```

Comment:
these first lines are test run related command and should not be in a create config file method